data-collection-scripts/ltc-scraper/Utah_Data_Extract.py

```diff
@@ -0,0 +1,47 @@
+import json
+from datetime import datetime
+from pytz import timezone
+from urllib.request import urlopen
+
+def load_json():
+    url = ("https://services6.arcgis.com/KaHXE9OkiB9e63uE/ArcGIS/rest/services/COVID19_Long_Term_Care_Facility_Impacts/FeatureServer/273/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=*&returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=true&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pjson&token=")
+    response = urlopen(url)
+    if response.getcode() == 200:
+        source = response.read()
+        dict_data = json.loads(source)
+    else:
+        print("Error")
+    #print(response.read())
+    #data = json.loads(response.read())
+
+    #f = open("Facilities_with_Active_Outbreaks.json", "r")
+    #data = f.read()
+    CSV_file = open("CSV_Data_Extract.csv", "w")
+
+    if not dict_data is None:
+        features_list = dict_data["features"]
+        for item in features_list:
+            fmt = datetime.today().strftime('%Y%m%d')
+            now_time = datetime.now(timezone('US/Eastern'))
+            line_data = now_time.strftime(fmt)
+            line_data += "," + "UT"
```

Comment:
Please don't output CSV data by trying to build it up from scratch this way. This will break the file format if any of the data fields contains a comma, newline, or anything else that needs to be escaped. The easiest thing to do is to transform the data into a Python array-of-dictionaries and use [Python's csv library](https://realpython.com/python-csv/#writing-csv-file-from-a-dictionary-with-csv) to write it out, which will take care of outputting a header with the column names, formatting everything, and escaping the data safely for you. Something more-or-less like (untested):

```si = StringIO()
writer = csv.writer(si, fieldnames=["Date Collected ", "State", "blank"])
writer.writeheader()
writer.writerow({'Date Collected': '20210112', 'State': 'UT', 'blank': ''})
output = si.getvalue()
print(output)
```